tools/custom-roles-analyzer/README.md

```diff
@@ -33,24 +35,27 @@ It will print results in file which contains following fields:
 
 It provides results in CSV or JSON format. Default is CSV.
 
+<!--
```

Comment:
Why is this commented out?

Reply:
I am planning to move binding-analysis to different tool. At this time just removing this from documentation and later will move the code.